### DISCUSSION TITLE
Can we set the values for the Y axis, to harmonize a series of curves ?

### DISCUSSION THREAD

**Discussion by liothub** [apexcharts/apexcharts.js#3990] [General]
Hello,

I am retrieving data from OpenWeatherMap Api (forecast 5 days) and  I create a curve for each day of the serie.
Then, the range of each serie (on the Y axis) seems determined by the actual data in the related serie and then they are uneven.
Can I force the range for the Y axis to be the same for the 6 curves? 
For example, the end of curve one (21°C) is at the same level that the start of curve two (18°C), taht hurts a bit.... the same for curves 5 & 6.

Thanks

![apex-range](https://github.com/apexcharts/apexcharts.js/assets/19572679/99f1024c-0b33-4b1f-b4af-9cdb94b5f31a)

**Comment by junedchhipa** [comment]
You can set the min and max of all charts to the same to harmonize.
Try this for all your charts
```javascript
yaxis: {
  min: 0,
  max: 50
}
```

**Reply by liothub** [reply]
Thank you so much.......    😀
